File: Vetor.c
// O algoritmo mostra a media da turma e também quais foram os alunos que ficaram acima da media;
// utilizando variáveis com VETORES;
#include <stdio.h>
main() //  metodo princial;
{
float notas[5];// variável do tipo real com VETORES;
float CalcMedia, SomaMedia; // variável do tipo real;
int contador; // variável do tipo interio;
contador = 0;
printf("Digite o valor da nota  1: "); //definição da n1
scanf("%f",&notas[1]);
printf("Digite o valor da nota  2: "); //definição da n2
scanf("%f",&notas[2]);
printf("Digite o valor da nota  3: "); //definição da n3
scanf("%f",&notas[3]);
printf("Digite o valor da nota  4: "); // definição da n2
scanf("%f",&notas[4]);
printf("Digite o valor da nota  5: "); //definição da n3
scanf("%f",&notas[5]);

SomaMedia = notas[1] + notas[2] + notas[3] + notas[4] + notas[5]; // calculo da soma das notas;
CalcMedia = SomaMedia / 5; // media da turma;

if ( notas[1] >= CalcMedia)//condição para ficar acima da media;
        contador++;
if ( notas[2] >= CalcMedia)// condiçãos para ficar aciama da media;
        contador++;
if ( notas[3] >= CalcMedia)//condiçãos para ficar aciama da media;
        contador++;
if ( notas[4] >= CalcMedia)//condiçãos para ficar aciama da media;
        contador++;
if ( notas[5] >= CalcMedia)// condiçãos para ficar aciama da media;
        contador++;        
printf("a media da turma e %f",CalcMedia); // saidas;
printf("e ficaram acima da media %d",contador);// saidas;
printf("\n\n>>>>>>>>>>>>>>>>>>>>>>>fim de Sistema<<<<<<<<<<<<<<<<<<<<<<<<<<");
}

File: Vetor.java
// O algoritmo mostra quantos alunos firam acima da media;
// utilizandos o vetor da varivel notas;



package JAVA;
import javax.swing.JOptionPane; // biblioteca para caixa de texto;
public class Vetor{
    public static void main(String args []){//medoto principal;
        
        float notas [] = new float [5]; // veriável na qual eu vou usar o vetor[] do tipo Real;
        float SomaNotas, CalcMedia;// para saber a media da turma; 
        int contador;// para ser feita a condição;
        
        contador = 0;
        // As variáveis que possui vetores devem são iniviadas aparte do 0 em JAVA; 
        notas[0] = Float.parseFloat(JOptionPane.showInputDialog("Digidi a nota 1: ")); // definição da nota 1
        notas[1] = Float.parseFloat(JOptionPane.showInputDialog("Digite a nota 2: ")); // definiação da nota 2
        notas[2] = Float.parseFloat(JOptionPane.showInputDialog("Digite a nota 3: ")); // definição da nota 3
        notas[3] = Float.parseFloat(JOptionPane.showInputDialog("Digite a nota 4: "));// denfinição da nota 4
        notas[4] = Float.parseFloat(JOptionPane.showInputDialog("Digite a nota 5: ")); // definição da nota 5

        SomaNotas = notas[0] + notas[1] + notas[2] + notas[3] + notas[4]; // soma das notas;
        CalcMedia = SomaNotas / 5; // media dos Alunos;

            if (notas[0] > CalcMedia) // condição;
            contador++;
            if (notas[1] > CalcMedia) //// condição;
            contador++;
            if (notas[2] > CalcMedia)// condição;
            contador++;
            if (notas[3] > CalcMedia)// condição;
            contador++;
            if (notas[4] > CalcMedia)// condição;
            contador++;

            JOptionPane.showMessageDialog(null,"a media da turma foi de"  +CalcMedia+
                                           "\n"+contador+" ficaram acima da media");// saídas;
    }
}